packages/mobile/src/identity/privateHashing.ts

```diff
@@ -79,7 +80,7 @@ function* doFetchPhoneHashPrivate(e164Number: string) {
     account,
     selfPhoneHash
   )
-  yield put(updateE164PhoneNumberSalts({ [e164Number]: details.salt }))
+  yield put(updateE164PhoneNumberSalts({ [e164Number]: details.pepper }))
```

Comment:
Is it possible to rename this? I assume we'll need to reference both for backward compatibility until everyone has upgraded to the latest version

Reply:
@codyborn Definitely, i plan handle the remaining 'salt' references in mobile next. There's quite a few, would have bloated this PR too much